src/hearings/models/hearings.enum.ts

```diff
@@ -134,7 +134,8 @@ export enum HearingDatePriorityConstEnum {
   MinHours = 0,
   MaxHours = 6,
   TotalMinMinutes = 5,
-  TotalMaxMinutes = 360,
+  TotalMaxMinutes = 55,
```

Comment:
what is the TotalMaxMinutes = 55 for?

Reply:
On the minutes field user can enter 0-55 only and the total minutes count shouldn't exceed 360 i.e. 6 hrs. 
If the user enters 1hr 10min then it will be considered and stored as 70.
If the user enters 0hr 70min it will not be considered valid data.